helmclient.go

```diff
@@ -430,14 +432,26 @@ func (c *Client) GetReleaseHistory(ctx context.Context, releaseName string) (*Re
 	{
 		release := resp.Releases[0]
 
+		var appVersion string
 		var version string
 		if release.Chart != nil && release.Chart.Metadata != nil {
+			appVersion = release.Chart.Metadata.AppVersion
 			version = release.Chart.Metadata.Version
 		}
 
+		var lastDeployed time.Time
+		if release.Info != nil {
+			lastDeployed, err = ptypes.Timestamp(release.Info.LastDeployed)
+			if err != nil {
+				return nil, microerror.Mask(err)
+			}
+		}
+
 		history = &ReleaseHistory{
```

Comment:
FMI what is the term history about? Name and version are not so much historical, isn't it?

Reply:
The naming is aligned with the upstream helm method that we call.

`c.newHelmClientFromTunnel(t).ReleaseHistory(releaseName, helmclient.WithMaxHistory(1))`